### PR TITLE
Cookies. Login adds a cookie and logout removes a cookie.

### PR DESCRIPTION
**Pull request by BenNewman90**
Made quite a few changes.

The cookieValidation function and cookieValidationHandler need completely re-writing.

### CODE REVIEW

File: client/src/helpers/validateCookie.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// const validateCookie = () => {
```
Comment (ghassanmas) [inline client/src/helpers/validateCookie.js]: Do you need this file anymore ?

Comment (BenNewman90) [reply client/src/helpers/validateCookie.js]: We will do. It just needs changing.

Comment (BenNewman90) [reply client/src/helpers/validateCookie.js]: now working on it.

Comment (ghassanmas) [reply client/src/helpers/validateCookie.js]: you will push on this branch/PR or the next one ?

Comment (BenNewman90) [reply client/src/helpers/validateCookie.js]: next

**Pull request merged**